app/components/team/Index.js

```diff
@@ -93,16 +94,16 @@ class TeamIndex extends React.PureComponent {
         <div>
           <Panel>
             <Panel.Section className="max-width-3">
-              <p>Teams allows you to group people, pipelines and permissions together.</p>
+              <p>Teams allows you to assign permissions to different groups of users and pipelines. You can create secret teams which are only visible to account admins and team members, or public teams that are visible to the your entire Buildkite organization.</p>
               <p>
                 <img
-                  src="//placekitten.com/800/480"
-                  width="400"
-                  height="240"
-                  alt=""
-                  title="This tiny kitten uses Teams - shouldn't you?"
+                  src={previewImage}
```

Comment:
honestly I’d just do this as an inline `require` call, just seems more obvious what’s going on that way